### PR TITLE
Generate case summaries based on aggregations.

### PR DESCRIPTION
**Pull request by rwiker**
Done as part of the  `_next_batch` method of class DocumentCollection.

### CODE REVIEW

File: src/fmu/sumo/explorer/objects/case.py [hunk 1]
```diff
@@ -16,8 +16,10 @@
 class Case(Document):
     """Class for representing a case in Sumo"""
 
-    def __init__(self, sumo: SumoClient, metadata: Dict, pit: Pit = None):
+    def __init__(self, sumo: SumoClient, metadata: Dict, summary: Dict,
```
Comment (perolavsvendsen) [inline src/fmu/sumo/explorer/objects/case.py:19]: Breaking change?

Comment (perolavsvendsen) [inline src/fmu/sumo/explorer/objects/case.py:19]: The name `summary` may be confusing, as the term is quite established for some specific data produced by simulators. Other synonyms that can be considered?

**Comment by rwiker** [comment]
Probably not a breaking change: the Case constructor is only called from CaseCollection.

**Comment by rwiker** [comment]
“sommaire”?

… or “overview”, which is the term used in https://github.com/equinor/fmu-sumo/issues/256.

**Pull request merged**